src/components/table-transactions/table-transactions.js

```diff
@@ -7,12 +9,34 @@ export class TableTransactionsComponent {
   }
 
   querySelectors() {
+    this.transactionPoint = this.mountPoint.querySelector(
+      ".table-transaction__tbody"
+    );
+  }
+
+  addStoredTransactions(list) {
+    let transHTML = "";
+    if (list) transHTML = list.map(data => templateRow({ row: data })).join("");
```

Comment:
please NEVER use `if` without `{` and `}`. in case you want to make code simpler just use ternary expressions
`const html = list  ? list.map(data => templateRow({ row: data })).join("") : ''`